### ISSUE TITLE
issue with freebitcoin

### ISSUE THREAD

**Issue by adelunreal** [seuntie900/DiceBot#81]
the bot don't start when connected to freebitco.in  https://prnt.sc/_tJyP4TPGTO1

**Comment by user39284** [comment]
Something changed on freebitco.in end. Dicebot latest version used to work fine, and now it's been giving this error for months. Freebitco.in support doesn't reply.

**Comment by TristanIvanov** [comment]
Yes, it seems that the app does not get an csrf token cookie after the login. Maybe it's  because of cloudflare or the hcaptcha... idk, unfortunatelly I suck at programming :(

**Comment by user39284** [comment]
All I know is their support said they switched from Google reCaptcha to hCaptcha, so maybe that broke it.
https://bitcointalk.org/index.php?topic=319540.msg59771021#msg59771021
They said they also tried messaging Seuntjie but he's been MIA for awhile now. Yeah betting on the website is not the same as using the bot :/
